### DISCUSSION TITLE
GOTV won´t start

### DISCUSSION THREAD

**Discussion by vic-951** [GameServerManagers/LinuxGSM#3820] [Q&A]
Hello there,

I tried to start gotv on csgo server from linuxgsm, i tried to put tv_enable 1 in every configuration file, but it won´t start. Now I installed a blank Server with linuxgsm and put 
`tv_enable 1
tv_port 7721
tv_delaymapchange 1
tv_delay 45
tv_deltacache 2
tv_dispatchmode 1
tv_maxclients 10
tv_maxrate 0
tv_overridemaster 0
tv_relayvoice 1
tv_snapshotrate 20
tv_timeout 60
tv_transmitall 1`
into ~/serverfiles/csgo/cfg/csgoserver.cfg and started the debug mode ./csgoserver d 
and I see only 
`tv_delay - 10
tv_delay1 - 15`
but thats not my config. How can I enable GOTV to record demos?

**Comment by h3o66** [comment]
seems like you need to do this manually, check this:

https://github.com/advancedfx/advancedfx/issues/199

**Comment by johnoclockdk** [comment]
@vic-951 can you show me your csgo configs maybe you missing some from there

**Comment by vic-951** [comment]
hey there @h3o66  sry but that link didn´t solve my problem. @johnoclockdk which config you want so see? there are many :)
